rs/test_utilities/execution_environment/src/lib.rs

```diff
@@ -2183,6 +2186,19 @@ impl ExecutionTestBuilder {
         self
     }
 
+    /// Note: This fails if the provided CyclesAccountManagerConfig has a different subnet size than self.
+    pub fn with_cycles_account_manager_config(mut self, cfg: CyclesAccountManagerConfig) -> Self {
```

Comment:
Why do we need to override the cycles account manager in tests?

Reply:
you could probably leave out this test as there's no such test for the remaining system APIs either (cost_http_request etc.), but I'll leave it up to you

Reply:
another reason why this test might not be very strong: by setting all costs to zero except for those you care about, you're masking off any potential other factors that could contribute to the cost of the call, but you don't consider them in your formula